sessiontxn/isolation/base.go

```diff
@@ -222,6 +223,12 @@ func (p *baseTxnContextProvider) ActivateTxn() (kv.Transaction, error) {
 		return nil, err
 	}
 
+	if p.constStartTS != 0 {
```

Comment:
How about making it part of the `prepareTxn` function?

Reply:
`WarmUp` will call `prepareTxn` , if we move it to `prepareTxn` it will still cost some cpu to when `WarmUp`

Reply:
Would an extra `TsFuture` be processed in the `prepareTxn` function, as next `replaceTxnTsFuture` seems to `replace` it?

Reply:
Yes. replaceTxnTsFuture will force replace the txn future, the caller will ensure the correctness. Because `replaceTxnTsFuture` is a private method which can only be called by provider, it is not a big problem